### PR TITLE
Desktop version branch

### PR DESCRIPTION
**Pull request by ReemMohamedAbdelfatah**
Add desktop version

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -3,571 +3,1197 @@ html body {
...
+/* *************************************************************** */
+
+/* *************************************************************** */
+
+/* ************************************************************** */
+
+@media only screen and (min-width: 765px) {
+  .mobile-version {
+    display: none;
+  }
```
Comment (codepantha) [inline style.css]: - [x] For the desktop version, please ensure that the `min-width` is between 768px or wider 👍

File: index.html [hunk 2]
```diff
@@ -1,299 +1,731 @@
...
-        <header >
-           <div class="nav-container">
+      <div class="top-header">
+        <header>
+          <div class="nav-container">
             <a>
-            <div id="logo">
-              Reem Muhammed
-            </div>
+              <div id="logo">Reem Muhammed</div>
```
Comment (codepantha) [inline index.html]: - [x] Please ensure to remove the horizontal scrolling as it negatively impacts user experience.
**NB:** I found that adding `overflow-x: hidden;` to the `body` element in your CSS can get rid of this.
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @ReemMohamedAbdelfatah 👋, 

You have done a great job so far👏 but there are some changes which you need to make in order to move to get approved. 😎



## Highlights 🎯 
✔️ Used correct GitHub Flow
✔️ No linter errors
✔️ Good commit history

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @ReemMohamedAbdelfatah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on implementing the required changes! 🎉👏🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**